src/main/java/net/atos/zac/mail/MailService.java

```diff
@@ -266,20 +244,36 @@ private String resolveZaakVariabelen(final String target, final Zaak zaak) {
             }
         }
 
-        if (resolvedTekst.contains(getVariabele(MailTemplateVariabelen.ZAAKTYPE))) {
+        if (resolvedTekst.contains(MailTemplateVariabelen.ZAAKTYPE.getVariabele())) {
             final Zaaktype zaaktype = ztcClientService.readZaaktype(zaak.getZaaktype());
             resolvedTekst = replaceVariabele(resolvedTekst, MailTemplateVariabelen.ZAAKTYPE, zaaktype.getOmschrijving());
         }
 
-        if (resolvedTekst.contains(getVariabele(MailTemplateVariabelen.INITIATOR))) {
+        if (resolvedTekst.contains(MailTemplateVariabelen.INITIATOR.getVariabele())) {
             final Rol<?> initiator = zgwApiService.findInitiatorForZaak(zaak);
             if (initiator != null) {
```

Comment:
Stop deze logica in een aparte method. Zo ook voor de code voor het initiator adres. Dit is namelijk wel een heel grote methode geworden

Reply:
Aangepast